core/src/components/range/range.tsx

```diff
@@ -185,14 +190,10 @@ export class Range implements ComponentInterface {
    */
   @Prop({ mutable: true }) value: RangeValue = 0;
   @Watch('value')
-  protected valueChanged(value: RangeValue) {
+  protected valueChanged() {
     if (!this.noUpdate) {
       this.updateRatio();
     }
-
-    value = this.ensureValueInBounds(value);
```

Comment:
Does this change mean the `ensureValueInBounds` will no longer be called if the developer passes a bad value? If so, we should call that out in the breaking changes guide.